### PR TITLE
fetch() is on BackgroundFetchManager not BackgroundFetch

### PR DESCRIPTION
**Pull request by teoli2003**
Noticed this flaw while reading another PR. Easy to fix.

### CODE REVIEW

File: files/en-us/web/api/backgroundfetchmanager/get/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ get(id)
 ### Parameters
 
 - `id`
-  - : the ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistration.fetch","fetch()")}}.
+  - : the ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistrationManager.fetch","fetch()")}}.
```
Comment (teoli2003) [inline files/en-us/web/api/backgroundfetchmanager/get/index.md:23]:
```suggestion
  - : The ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistrationManager.fetch","fetch()")}}.
```

File: files/en-us/web/api/backgroundfetchmanager/get/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ get(id)
 ### Parameters
 
 - `id`
-  - : the ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistration.fetch","fetch()")}}.
+  - : The ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistrationManager.fetch","fetch()")}}.
```
Comment (wbamberg) [inline files/en-us/web/api/backgroundfetchmanager/get/index.md:23]:
```suggestion
  - : The ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchManager.fetch","fetch()")}}.
```

**Comment by teoli2003** [comment]
Not awake yet, sorry!

**Review by wbamberg — APPROVED** [review]
👍

**Pull request merged**